taskfiles/toolchains.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+tasks:
+  rust:
+    run: "once"
+    cmds:
+      # Installs Rust toolchains using rustup. If the toolchains are already installed, this task
+      # will ensure they are updated to the latest versions.
+      - |-
+        curl --proto "=https" --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
* Why not add `--default-toolchain none` like the docs suggest?
* We usually prefer using long-form options rather than short-forms (unless that affects compatibility on macOS).